src/htm/algorithms/SpatialPooler.hpp

```diff
@@ -133,22 +133,6 @@ class SpatialPooler : public Serializable
         most N columns remain ON within a local inhibition area, where
         N = localAreaDensity * (total number of columns in inhibition
         area). 
-	If localAreaDensity is set to any value less than  0, 
-	output sparsity will be determined by the numActivePerInhArea.
-
-  @param numActiveColumnsPerInhArea An alternate way to control the sparsity of
-        active columns. When numActivePerInhArea > 0, the inhibition logic will insure that
-        at most 'numActivePerInhArea' columns remain ON within a local
-        inhibition area (the size of which is set by the internally
-        calculated inhibitionRadius). When using this method, as columns
-        learn and grow their effective receptive fields, the
-        inhibitionRadius will grow, and hence the net density of the
-        active columns will *decrease*. This is in contrast to the
-        localAreaDensity method, which keeps the density of active
```

Comment:
I think localAreaDensity should be used instead of numActiveColumnsPerInhArea, as it keeps const sparsity.